install/0000_50_machine-api-operator_01_images.configmap.yaml

```diff
@@ -4,4 +4,4 @@ metadata:
   name: machine-api-operator-images
   namespace: openshift-cluster-api
 data:
-  images.json: '{"clusterAPIControllerAWS": "registry.svc.ci.openshift.org/openshift/aws-machine-controllers:crd", "clusterAPIControllerManagerAWS": "registry.svc.ci.openshift.org/openshift/aws-machine-controllers:crd", "clusterAPIControllerOpenStack": "docker.io/openshift/origin-openstack-machine-controllers:v4.0.0", "clusterAPIControllerManagerOpenStack": "docker.io/openshift/origin-openstack-machine-controllers:v4.0.0", "clusterAPIControllerManagerLibvirt": "docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0", "clusterAPIControllerLibvirt": "docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0"}'
+  images.json: '{"clusterAPIControllerAWS": "registry.svc.ci.openshift.org/openshift/aws-machine-controllers:crd", "clusterAPIControllerManagerAWS": "registry.svc.ci.openshift.org/openshift/aws-machine-controllers:crd", "clusterAPIControllerOpenStack": "docker.io/openshift/origin-openstack-machine-controllers:v4.0.0", "clusterAPIControllerManagerOpenStack": "docker.io/openshift/origin-openstack-machine-controllers:v4.0.0", "clusterAPIControllerManagerLibvirt": "registry.svc.ci.openshift.org/openshift/libvirt-machine-controllers:crd", "clusterAPIControllerLibvirt": "registry.svc.ci.openshift.org/openshift/libvirt-machine-controllers:crd"}'
```

Comment:
Here as well: https://github.com/openshift/machine-api-operator/blob/3a9961e8bf8e616f9db233607a434f57679065a4/tests/e2e/manifests/images.configmap.yaml#L10-L11

Reply:
thanks! fyi that does not have any effect, libvirt is not being tested here. We need to add framework like testing in this repo